src/content/app/genome-browser/components/drawer/drawer-views/regulation-legend/RegulationLegend.tsx

```diff
@@ -16,7 +16,7 @@
 
 import React from 'react';
 
-import regulationLegends from 'src/content/app/genome-browser/constants/regulationLegends';
+import regulationLegend from 'src/content/app/genome-browser/constants/regulationLegend';
```

Comment:
![image](https://github.com/Ensembl/ensembl-client/assets/6834224/c3c77406-d6a8-4d9e-94f4-515d1bb2e0fb)

Should rename the file as well